src/network/netlink.rs

```diff
@@ -212,16 +212,22 @@ impl Socket {
         Ok(())
     }
 
-    fn create_route_msg(route: &Route) -> RouteMessage {
+    fn create_route_msg(route: &Route, metric: Option<u32>) -> RouteMessage {
```

Comment:
should we make this part of the route type instead of adding new parameters?
this will break @baude's work in the dhcp proxy, since he also reuses the route and addr calls there

Reply:
Sure, I can change this